custom-transformer-2.0_2.11/src/main/scala/org/apache/spark/ml/HasRawInputCol.scala

```diff
@@ -0,0 +1,12 @@
+package org.apache.spark.ml
+
+import org.apache.spark.ml.param.{Param, Params}
+
+/**
+  * Created by vinay.varma on 08/11/16.
```

Comment:
-please update the author at all places.

Reply:
done.